docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,7 +110,52 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Updating and Retreiving Playlist
```

Comment:
Suggestion. As new section is added, how about adding a simple description sentence or words about it in the beginning of this page, as other sections do?

As it has similarity to "Updating and Retrieving Information", it would be possible to be located near list item in line 8, or can be merged into it. In that case, this section also would be better to go between "Updating and Retrieving Information" and "Sending and Processing Commands" sections.

Reply:
I just added this section like Native Media Controller guideline.